src/search/search.tsx

```diff
@@ -1,37 +1,22 @@
 import React from 'react'
 
-import SearchResults from './result'
 import QuerySymbols from './querySymbols'
+import SearchResults from './result'
 
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 
 import './search.css'
+import { Company, SearchProps, SearchState } from './types';
```

Comment:
is SearchState a local Compomnent state that is used only by the component? If so, you may just place it here.